bin/testEnvLocalK8S.sh

```diff
@@ -63,7 +64,7 @@ function getDeps() {
      curl -Lo ${TOP}/bin/kube-apiserver https://storage.googleapis.com/kubernetes-release/release/v1.7.4/bin/linux/amd64/kube-apiserver && chmod +x ${TOP}/bin/kube-apiserver
    fi
    if [ ! -f $TOP/bin/etcd ] ; then
-     go get github.com/coreos/etcd/cmd/etcd
+     curl -L https://github.com/coreos/etcd/releases/download/${ETCD_VERSION}/etcd-${ETCD_VERSION}-linux-amd64.tar.gz | tar xz -O etcd-${ETCD_VERSION}-linux-amd64/etcd > ${TOP}/bin/etcd && chmod +x ${TOP}/bin/etcd
```

Comment:
I have a feeling that etcd is installed in our ci image.. Check the Dockerfile in .circleci/Dockerfile

Reply:
yeah, but there were two problems:
- we want to be able to run `make localTestEnv` from machines that don't have etcd installed
- the install path used in the docker image is also wrong.  there is no `github.com/coreos/etcd/cmd/etcd` anymore.   it appears they recently shuffled things around in their repo and lifted that `main` to the repo root.